app/assets/javascripts/atlases/show.js.erb

```diff
@@ -163,7 +163,7 @@ atlasJS.appendPlaceMetadata = function( panel, place ) {
     panel.append( addPlaceLink );
     panel.append( "<br>" );
   }
-  if( !existing || ( existing.type != "exploded"  && existing.place.admin_level < 2 ) ) {
+  if( !existing || ( existing.type != "exploded"  && existing.place.admin_level < 20 ) ) {
```

Comment:
Since this is an ERB can we just refer to `Place::COUNTY_LEVEL`? Or are we trying to avoid model references in the JS templates?

Reply:
Yeah I think we probably could use the model constant here. I'll try that out